### PR TITLE
Better phrasing around 1.1.0

### LINKED ISSUE #8129: openssl/INSTALL makes numerous references to version 1.1.0 despite being 1.1.1
Some help files seem to still be out of date for 1.1.1 and and reference 1.1.0. A quick code grep should be done for "1.1.0"/"1.1.x" to see if any other locations need to be changed.

### PR DESCRIPTION
**Pull request by levitte**
Fixes #8129

### CODE REVIEW

**Review by mattcaswell — APPROVED** [review]

File: INSTALL [hunk 1]
```diff
@@ -973,10 +973,10 @@
 
   *  COMPILING existing applications
 
-     OpenSSL 1.1.0 hides a number of structures that were previously
-     open.  This includes all internal libssl structures and a number
-     of EVP types.  Accessor functions have been added to allow
-     controlled access to the structures' data.
+     From version 1.1.0 and on, OpenSSL hides a number of structures
```
Comment (mspncp) [inline INSTALL]: Alternative:
```
Starting with version 1.1.0, OpenSSL ...
```

Comment (levitte) [reply INSTALL]: Fixed

**Review by mspncp — APPROVED** [review]
LGTM

**Comment by levitte** [comment]
Merged.

master:
62b563b9df161a992fde18a0cb0d1a0969158412 Better phrasing around 1.1.0

1.1.1:
822e6d95e063a8fb6cd6b8e272dc5d9338c72601 Better phrasing around 1.1.0

**Pull request closed without merging**